oneflow/python/test/ops/test_sort.py

```diff
@@ -57,10 +58,10 @@ def SortJob(
 def gen_arg_list():
     arg_dict = OrderedDict()
     arg_dict["device_type"] = ["cpu", "gpu"]
-    arg_dict["in_shape"] = [(100,), (100, 100), (10, 10, 200)]
+    arg_dict["in_shape"] = [(100,), (10, 10, 200)]
```

Comment:
sort & argsort跑起来还是比较耗时的，建议尽可能较少笛卡尔积和输入Tensor大小。
```suggestion
    arg_dict["in_shape"] = [(10,), (10, 10, 20)]
```